### PR TITLE
Added fpdf writer

### PR DESCRIPTION
**Pull request by MaximilianKresse**
See #256

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -21,11 +21,13 @@
...
     "require-dev": {
-        "endroid/quality": "dev-master"
+        "endroid/quality": "dev-master",
+        "setasign/fpdf": "^1.8"
     },
     "suggest": {
         "roave/security-advisories": "Avoids installation of package versions with vulnerabilities",
-        "symfony/security-checker": "Checks your composer.lock for vulnerabilities"
+        "symfony/security-checker": "Checks your composer.lock for vulnerabilities",
+        "setasign/fpdf": "Required to use the QR FPDF writer."
```
Comment (JanSlabon) [inline composer.json:30]: Should be:
> "setasign/fpdf": "Required to use the FPDF writer."

File: src/Writer/FpdfWriter.php [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+     *
+     * Allowed values: 'mm', 'pt', 'cm', 'in'
+     */
+    public const WRITER_OPTION_MEASURE_UNIT = 'fpdf_measure_unit';
+
+    public function __construct()
+    {
+        if(!\class_exists(\FPDF::class)){
+            throw new \BadMethodCallException(
+                'The QRFpdf output requires FPDF as dependency but the class "\FPDF" couldn\'t be found.'
```
Comment (JanSlabon) [inline src/Writer/FpdfWriter.php:31]: > This writer requires FPDF as dependency but the class "\FPDF" couldn\'t be found.

Comment (MaximilianKresse) [reply src/Writer/FpdfWriter.php]: Placed by "The Fpdf writer requires FPDF as dependency but the class "\\FPDF" couldn\'t be found." instead.

**Comment by endroid** [comment]
Thank you!

**Pull request merged**